src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  margin-right: $card-margin-right;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-child(-n+4) {
+    margin-bottom: 44px;
+  }
+
+  &__img {
+    display: flex;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: $card-font-size_medium;
+    line-height: $card-height-high;
+    color: $color-dark-text;
+  }
+
+  &__product-code {
+    font-weight: $card-font-weight-thin;
+    line-height: $card-height-medium;
+    color: $color-grey-text;
+    margin-top: 4px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-weight: $card-font-weight-thin;
+    margin-top: 16px;
+  }
+
+  &__reviews-count {
+    color: $color-dark-text;
+    line-height: $card-height-medium;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 24px;
+    
+  }
+
+  &__price-name {
+    font-weight: $card-font-weight-thin;
+    font-size: $card-font-size_medium;
+    line-height: $card-height-high;
+    color: $color-grey-text;
+  }
+
+  &__price-value {
+    font-weight: $card-font-weight-bold;
+    font-size: 16px;
+    line-height: $card-height-high;
+    color: $color-dark-text;
+  }
+
+  &__link {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-top: 16px;
+    padding: 12px;
+    border-radius: 5px;
+    font-weight: $card-font-weight-bold;
+    font-size: 14px;
+    line-height: 16px;
+    text-decoration: none;
+    text-transform: uppercase;
+    border: 1px solid #fff;
+    background: $card-color-blue;
+    color: $card-color-button-white;
+    transition: color $transition-duration,
+      background-color $transition-duration;
+  }
+
+  &__link:hover {
+    color: $card-color-blue;
+    background-color: $card-color-button-white;
+    border-color: var(--section-card-color-button-blue);
```

Comment:
why do we need to change border-color? just move it to `link` like
`border: 1px solid var(--section-card-color-button-blue);`

Reply:
Hi, this remained from the condition of the previous task: (Layout_product_card)
9. [STYLES] - Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover.
I'll fix this.